### DISCUSSION TITLE
How to get a URL for a page in current language in template?

### DISCUSSION THREAD

**Discussion by dtluna** [wagtail/wagtail-localize#365] [Ideas]
The website I'm developing has an "About" page in multiple languages. What kind of template code would I need to add a URL to the current language version of the About page?

```htmldjango
{% slugurl 'about' %}
```
doesn't cut it because it doesn't respect the language setting.

**Comment by kaedroho** [comment]
Typically, if you have a page object you can just put `.localized` on the end to convert it to the local language. But that's no possible for this tag. This is something that we should solve in the implementation of that tag in Wagtail itself. Perhaps by adding a new parameter to ``slugurl``: ``{% slugurl 'about' localized %}
